examples/lusid/instruments/upload-simple-instruments.sql

```diff
@@ -0,0 +1,32 @@
+-- ============================================================
+
+-- 1. In this query, we run an ETL process on some instruments.
+-- 2. First, we load a CSV file of instruments from Drive.
+-- 3. Next, we transform the shape of the instrument data.
+-- 4. Finally we upload the instrument data into LUSID.
+
+-- ============================================================
+
+-- Load file of instruments data
+@instruments_data =
+
+use Drive.Excel
+--file=/luminesce-examples/instruments.xlsx
+--worksheet=simple_instruments
+enduse;
```

Comment:
Simple Instruments are used to book instruments we can't book with traditional types and so for the example we should use something to represent that. I have slacked you an updated excel sheet with the simple instruments as shopping centres, could you use that?

It is all the same fields so you should be able to just update the spreadsheet and not need to update your query.

Reply:
Excel file changed to shopping centre data.